daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -182,8 +185,8 @@ enum PrimType {
   // Builtin type 'Int64'
   INT64 = 2;
 
-  // Builtin type 'Int64'
-  DECIMAL = 3;
+  // Builtin type 'Numeric'
```

Comment:
This is a type for older versions of DAML-LF but a type constructor for newer versions. We should document this.

Reply:
I update the daml-lf spec (could you please take a look?) and add a note here that it was called ``DECIAML`` is previous version.